packages/cashier/src/components/cashier-locked/cashier-locked.tsx

```diff
@@ -111,7 +125,13 @@ const CashierLocked = ({
                 <Localize
                     i18n_default_text='Please complete the <0>Appropriateness Test</0> to access your cashier.'
                     components={[
-                        <a key={0} className='link' rel='noopener noreferrer' href={'/account/financial-assessment'} />,
+                        <a
+                            key={0}
+                            className='link'
+                            rel='noopener noreferrer'
+                            href={'/account/financial-assessment'}
+                            data-testid='financial-assessment-link'
```

Comment:
Could you update `data-testid`s base on this https://wikijs.deriv.cloud/Frontend/General-information/test_guide#:~:text=Make%20sure%20you%20apply%20correct%20names%20while%20using%20data%2Dtestid%20attribute.%20It%20is%20required%20to%20use%20snake_case%20and%20dt%20prefix%3A

Reply:
Changed.